manifest/provider/server.go

```diff
@@ -108,3 +108,27 @@ func (s *RawProviderServer) MoveResourceState(ctx context.Context, req *tfprotov
 	resp := &tfprotov5.MoveResourceStateResponse{}
 	return resp, nil
 }
+
+func (s *RawProviderServer) OpenEphemeralResource(ctx context.Context, req *tfprotov5.OpenEphemeralResourceRequest) (*tfprotov5.OpenEphemeralResourceResponse, error) {
```

Comment:
Had to add these stubs because Updating the SDK dependencies brought in this interface.